src/transformers/models/auto/modeling_auto.py

```diff
@@ -827,6 +828,16 @@
     ]
 )
 
+BACKBONE_MAPPING_NAMES = OrderedDict(
+    [
+        # Model for Instance Segmentation mapping
+        ("maskformer-swin", "MaskFormerSwinBackbone"),
+        ("resnet", "ResNetBackbone"),
+        ("swin", "MaskFormerSwinBackbone"),  # for backward compatibility
```

Comment:
This does not make any sense since we could have a `SwinBackbone` one of these days.